src/helpers-test.js

```diff
@@ -85,3 +85,35 @@ QUnit.test(".getIndex should not sort unchanged items #33", function() {
 	QUnit.equal(res3, 2);
 	QUnit.equal(res4, 3);
 });
+
+QUnit.test("Missed schema on helper.getIndex #45", function() {
+	var items = [
+		{id: 1, name: "Item 0"},
+		{id: 2, name: "Item 1"},
+		{id: 3, name: "Item 2"},
+		{id: 4, name: "Item 3"},
+		{id: 5, name: "Item 4"}
+	];
+
+	canReflect.eachIndex(items, function(item) {
+		canReflect.assignSymbols(item, {
```

Comment:
the items should not have a schema right?

Reply:
@justinbmeyer I updated the test!